src/viser/client/src/WebsocketInterface.tsx

```diff
@@ -598,6 +598,30 @@ function useMessageHandler() {
         return;
       }
 
+      // Add GlTF Message
+      case "GlTFMessage": {
+        const binaryData = atob(message.gltf_base64_data);
+        const arrayBuffer = new ArrayBuffer(binaryData.length);
+        const uint8Array = new Uint8Array(arrayBuffer);
+
+        for (let i = 0; i < binaryData.length; i++) {
+          uint8Array[i] = binaryData.charCodeAt(i);
```

Comment:
It would be nice to avoid this copy if we move away from base64! Iterating over each byte in javascript seems expensive.